elements/pfe-tabs/src/pfe-tab-panel.scss

```diff
@@ -17,12 +17,21 @@ $LOCAL-VARIABLES: (
   ::slotted(*) {
     color: #{pfe-broadcasted(text)};
   }
-
   [tabindex] {
     display: flex;
     height: 100%;
   }
 
+ //updated saturated styles
+  @at-root #{&}([pfe-variant="earth"][on="saturated"]) .container{
+    background-color: #fff;
+    color: #000 !important;
```

Comment:
```suggestion
    color: pfe-var(text);
```

Reply:
I'm using the "text" color variable instead of "link" throughout these suggestions because the mockups treat the copy as if it is regular text instead of a link (there's no blue). Basically there is enough other design treatment around the copy to make it stand out as a UI element, thus it doesn't have to lean on the bright blue color to let users know it's clickable.